src/fcgen/fastcat_types.yaml

```diff
@@ -346,6 +346,19 @@ states:
       - name: adc_value_ch8
         type: int16_t
 
+  - name: ild1900
+    fields:
+    - name: distance
+      type: double
```

Comment:
Consider naming this `distance_m` to denote the fixed unit this sensor outputs. Should help head off questions in the future.

Reply:
I was hesitant about it because other devices do not always specify units.

My personal preference is to use unit postfix always whenever units are fixed. But that doesn't seem to be the standard currently used in the libraries.

I can add units to `distance` and `timestamp` here, in JSD, and the ROS message. Let me know.